string frame_id
mines/objective[] objective
mines/o_r_state[] o_r_state
mines/cross_trajectory[] cross_trajectory
mines/collective_trajectories collective_trajectories
string trigger
int8 num_agent_traj
float64 step_time
